witness/src/Dscp/Witness/AVL/Proof.hs

```diff
@@ -0,0 +1,17 @@
+-- | Avl+ related types and instances.
+
+module Dscp.Witness.AVL.Proof
+    ( AvlProof
+    ) where
+
+import Codec.Serialise (Serialise (..))
+import qualified Data.Tree.AVL as AVL
+import Snowdrop.Execution ()
+
+import Dscp.Snowdrop.Configuration (Ids, Values)
+import Dscp.Witness.AVL.Hash
+
+-- Should it be a newtype?
```

Comment:
I think it should be, so its instances will be more or less in the same place as instances of other things we put into the block.

Reply:
Is it not going to change?